signer/src/bin/demo_cli.rs

```diff
@@ -248,6 +255,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         CliCommand::Info => exec_info(&ctx).await?,
         CliCommand::FundBtc(args) => exec_fund_btc(&ctx, args).await?,
         CliCommand::FundStx(args) => exec_fund_stx(&ctx, args).await?,
+        CliCommand::Forward(args) => exec_forward(&ctx, args).await?,
```

Comment:
What is meant by `Forward`? Wondering if there's a more descriptive/obvious name..

Reply:
Renamed in [68f2d61](https://github.com/stacks-sbtc/sbtc/pull/1585/commits/68f2d61ae661cd29a87eae9c1ec7c1cbd9b9e6c2)